docs/config-file.md

```diff
@@ -21,13 +21,23 @@ export default async function () {
       httpServer: customServer, // A custom HTTP server of your own.
       adapter: 'native', // Default. Can also be 'socket.io' or a reference to a custom adapter.
       port: process.env.PORT,
+    },
+    cluster: {
+      adapter: 'redis',
+      name: 'cluster', // Default. Name of your cluster.
+      url: 'redis://localhost:6379', // Server URL used by adapter for clustering
     }
   }
 }
 ```
 
-|Field|Description|
-|---|---|
-|adapter|The Glee adapter to use for the WebSocket server. Defaults to a "native" WebSocket implementation. Other allowed values are `socket.io` (to use the [Socket.IO](https://socket.io/) Glee adapter) or a reference to a custom adapter.
-|httpServer|A custom HTTP server of your own. E.g., an [Express](https://expressjs.com/en/4x/api.html) server or any object that implements the [http.Server](https://nodejs.org/api/http.html#http_class_http_server) interface.
-|port|The port to use when binding the WebSocket server. This is useful when your server is behind a proxy and the port exposed for consumption is not the same as the port your application should be bound to. Defaults to the port specified in the selected AsyncAPI server.
+|Parent|Field|Description|
```

Comment:
You don't need this `Parent` column. Maybe you want to follow the dot notation as we're doing here: https://github.com/asyncapi/generator/blob/master/docs/api.md#new-generatortemplatename-targetdir-options.

So field names would be `websocket.adapter` instead of `adapter` and so on.